### PR TITLE
Added Export Values Unit Testcase

### PR DESCRIPTION
**Pull request by gokulmInfy**
Ready for Review

### CODE REVIEW

File: src/test/java/com/att/m2x/java/M2XClientTest.java [hunk 1]
```diff
@@ -316,6 +316,18 @@ public void deviceApiTest() throws IOException, InterruptedException
 		assertThat(response.status, is(200));
 		assertThat(response.json().getJSONArray("values").length(), is(1));
 
+        //export value
```
Comment (kristinpeterson) [inline src/test/java/com/att/m2x/java/M2XClientTest.java]: @gokulmInfy could you please move this line to align with the rest of the block?

**Comment by gokulmInfy** [comment]
Ready for Review

**Comment by majidabdul** [comment]
@gokulmInfy please work on the review comments.

**Comment by kristinpeterson** [comment]
@gokulmInfy @majidabdul looks great!

**Pull request merged**